inst/bench/bench.R

```diff
@@ -54,34 +53,18 @@ analyse_dt <- function(patients, window_start, window_end) {
 }
 
 build_graph <- function(left = -Inf, right = Inf) {
```

Comment:
can we maybe consider two graphs: one when only one feature is calculated and one for three features?